dali/test/python/test_operator_spectrogram.py

```diff
@@ -114,32 +115,33 @@ def iter_setup(self):
         self.feed_input(self.data, data)
 
 def check_operator_spectrogram_vs_python(device, batch_size, input_shape,
-                                         nfft, window_length, window_step):
+                                         nfft, window_length, window_step, center):
     eii1 = RandomDataIterator(batch_size, shape=input_shape, dtype=np.float32)
     eii2 = RandomDataIterator(batch_size, shape=input_shape, dtype=np.float32)
     compare_pipelines(
         SpectrogramPipeline(device, batch_size, iter(eii1), nfft=nfft, window=None,
-                            window_length=window_length, window_step=window_step),
+                            window_length=window_length, window_step=window_step, center=center),
         SpectrogramPythonPipeline(device, batch_size, iter(eii2), window=None, nfft=nfft,
-                                  window_length=window_length, window_step=window_step),
-        batch_size=batch_size, N_iterations=5, eps=1e-04)
+                                  window_length=window_length, window_step=window_step, center=center),
+        batch_size=batch_size, N_iterations=3, eps=1e-04)
```

Comment:
:+1:

Reply:
Perhaps it would be even faster to run CPU and GPU here and only run the Python thingy once.